src/Vocup/Forms/PracticeDialog.cs

```diff
@@ -154,10 +157,25 @@ public void LoadNextWord()
             TbCorrectAnswer.BackColor = SystemColors.Control;
         }
 
-        if (book.PracticeMode == PracticeMode.AskForForeignLang)
+        currentPracticeMode = book.PracticeMode;
+
+        if (currentPracticeMode == PracticeMode.AskForBothMixed)
+        {
+            if (random.Next(100) < 50)
```

Comment:
Since .NET 6 it is not required to create an instance of `Random` anymore.

```suggestion
            if (Random.Shared.Next(2) == 0)
```